[2021-12-21 22:28:10,411] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: data_pipeline.super_resolution scheduled__2021-12-19T00:04:00+00:00 [queued]>
[2021-12-21 22:28:10,415] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: data_pipeline.super_resolution scheduled__2021-12-19T00:04:00+00:00 [queued]>
[2021-12-21 22:28:10,415] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-21 22:28:10,415] {taskinstance.py:1242} INFO - Starting attempt 1 of 4
[2021-12-21 22:28:10,415] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-21 22:28:10,424] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): super_resolution> on 2021-12-19 00:04:00+00:00
[2021-12-21 22:28:10,425] {standard_task_runner.py:52} INFO - Started process 97109 to run task
[2021-12-21 22:28:10,427] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'data_pipeline', 'super_resolution', 'scheduled__2021-12-19T00:04:00+00:00', '--job-id', '86', '--raw', '--subdir', '/home/myamya/project/dockers/airflow/dags/hello_world.py', '--cfg-path', '/tmp/tmpqt94f0tk', '--error-file', '/tmp/tmp51j4ersv']
[2021-12-21 22:28:10,427] {standard_task_runner.py:77} INFO - Job 86: Subtask super_resolution
[2021-12-21 22:28:10,444] {logging_mixin.py:109} INFO - Running <TaskInstance: data_pipeline.super_resolution scheduled__2021-12-19T00:04:00+00:00 [running]> on host myamya
[2021-12-21 22:28:10,465] {taskinstance.py:1412} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=yangjae
AIRFLOW_CTX_DAG_ID=data_pipeline
AIRFLOW_CTX_TASK_ID=super_resolution
AIRFLOW_CTX_EXECUTION_DATE=2021-12-19T00:04:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-19T00:04:00+00:00
[2021-12-21 22:28:10,465] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2021-12-21 22:28:10,465] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'docker run -it -v /home/myamya/project/image_files/:/home/myamya/project/image_files/ sure:0.3 /bin/bash /home/myamya/project/init.sh ']
[2021-12-21 22:28:10,469] {subprocess.py:85} INFO - Output:
[2021-12-21 22:28:12,162] {subprocess.py:89} INFO - /opt/conda/lib/python3.8/site-packages/torch/cuda/__init__.py:52: UserWarning: CUDA initialization: Found no NVIDIA driver on your system. Please check that you have an NVIDIA GPU and installed a driver from http://www.nvidia.com/Download/index.aspx (Triggered internally at  /opt/conda/conda-bld/pytorch_1603729096996/work/c10/cuda/CUDAFunctions.cpp:100.)
[2021-12-21 22:28:12,162] {subprocess.py:89} INFO -   return torch._C._cuda_getDeviceCount() > 0
[2021-12-21 22:28:13,222] {subprocess.py:89} INFO - Downloading: "https://github.com/xinntao/facexlib/releases/download/v0.1.0/detection_Resnet50_Final.pth" to /opt/conda/lib/python3.8/site-packages/facexlib/weights/detection_Resnet50_Final.pth
[2021-12-21 22:28:13,222] {subprocess.py:89} INFO - 
[2021-12-21 22:28:16,127] {subprocess.py:89} INFO -   0% 0.00/104M [00:00<?, ?B/s]  0% 104k/104M [00:00<01:59, 915kB/s]  0% 392k/104M [00:00<01:34, 1.15MB/s]  1% 1.21M/104M [00:00<01:09, 1.56MB/s]  4% 3.88M/104M [00:00<00:48, 2.17MB/s]  9% 8.96M/104M [00:00<00:32, 3.05MB/s] 13% 13.8M/104M [00:00<00:22, 4.24MB/s] 18% 18.7M/104M [00:00<00:15, 5.86MB/s] 23% 23.6M/104M [00:00<00:10, 7.98MB/s] 27% 28.6M/104M [00:00<00:07, 10.7MB/s] 32% 33.5M/104M [00:01<00:05, 14.0MB/s] 37% 38.5M/104M [00:01<00:03, 18.0MB/s] 42% 43.4M/104M [00:01<00:02, 22.3MB/s] 46% 48.3M/104M [00:01<00:02, 26.9MB/s] 51% 53.3M/104M [00:01<00:01, 31.5MB/s] 56% 58.3M/104M [00:01<00:01, 35.7MB/s] 61% 63.2M/104M [00:01<00:01, 39.3MB/s] 65% 68.1M/104M [00:01<00:00, 42.4MB/s] 70% 73.1M/104M [00:01<00:00, 44.8MB/s] 75% 78.0M/104M [00:01<00:00, 46.6MB/s] 79% 82.9M/104M [00:02<00:00, 48.1MB/s] 84% 87.9M/104M [00:02<00:00, 49.2MB/s] 89% 92.9M/104M [00:02<00:00, 49.8MB/s] 94% 97.8M/104M [00:02<00:00, 50.3MB/s] 98% 103M/104M [00:02<00:00, 50.6MB/s] 100% 104M/104M [00:02<00:00, 44.5MB/s]
[2021-12-21 22:28:16,268] {subprocess.py:89} INFO - Downloading: "https://github.com/TencentARC/GFPGAN/releases/download/v0.2.0/GFPGANCleanv1-NoCE-C2.pth" to /opt/conda/lib/python3.8/site-packages/gfpgan/weights/GFPGANCleanv1-NoCE-C2.pth
[2021-12-21 22:28:16,268] {subprocess.py:89} INFO - 
[2021-12-21 22:28:23,956] {subprocess.py:89} INFO -   0% 0.00/332M [00:00<?, ?B/s]  0% 184k/332M [00:00<03:05, 1.88MB/s]  0% 1.01M/332M [00:00<02:21, 2.46MB/s]  1% 4.51M/332M [00:00<01:41, 3.38MB/s]  2% 8.07M/332M [00:00<01:13, 4.65MB/s]  4% 11.8M/332M [00:00<00:53, 6.33MB/s]  5% 15.1M/332M [00:00<00:39, 8.36MB/s]  6% 20.5M/332M [00:00<00:29, 11.2MB/s]  8% 25.4M/332M [00:00<00:22, 14.3MB/s]  9% 31.0M/332M [00:00<00:17, 18.5MB/s] 11% 36.6M/332M [00:01<00:13, 23.3MB/s] 12% 41.2M/332M [00:01<00:11, 26.7MB/s] 14% 46.4M/332M [00:01<00:09, 31.6MB/s] 16% 52.0M/332M [00:01<00:08, 36.6MB/s] 17% 56.9M/332M [00:01<00:07, 38.7MB/s] 19% 61.7M/332M [00:01<00:06, 41.6MB/s] 20% 67.3M/332M [00:01<00:06, 45.5MB/s] 22% 72.3M/332M [00:01<00:06, 43.8MB/s] 23% 77.8M/332M [00:01<00:05, 47.3MB/s] 25% 83.4M/332M [00:02<00:05, 50.1MB/s] 27% 88.5M/332M [00:02<00:05, 46.6MB/s] 28% 94.1M/332M [00:02<00:05, 49.7MB/s] 30% 99.1M/332M [00:02<00:05, 46.5MB/s] 31% 105M/332M [00:02<00:04, 49.3MB/s]  33% 110M/332M [00:02<00:04, 51.7MB/s] 35% 115M/332M [00:02<00:04, 48.9MB/s] 36% 120M/332M [00:02<00:04, 50.2MB/s] 38% 126M/332M [00:02<00:04, 52.4MB/s] 39% 131M/332M [00:03<00:04, 48.1MB/s] 41% 137M/332M [00:03<00:04, 50.9MB/s] 43% 142M/332M [00:03<00:03, 52.9MB/s] 44% 147M/332M [00:03<00:04, 48.5MB/s] 46% 153M/332M [00:03<00:03, 51.1MB/s] 48% 158M/332M [00:03<00:03, 48.1MB/s] 49% 163M/332M [00:03<00:03, 49.9MB/s] 51% 169M/332M [00:03<00:03, 52.2MB/s] 52% 174M/332M [00:03<00:03, 50.7MB/s] 54% 179M/332M [00:04<00:03, 49.8MB/s] 55% 184M/332M [00:04<00:02, 52.1MB/s] 57% 189M/332M [00:04<00:03, 47.8MB/s] 59% 195M/332M [00:04<00:02, 50.6MB/s] 60% 200M/332M [00:04<00:02, 47.3MB/s] 62% 206M/332M [00:04<00:02, 50.1MB/s] 64% 212M/332M [00:04<00:02, 52.3MB/s] 65% 217M/332M [00:04<00:02, 49.2MB/s] 67% 222M/332M [00:04<00:02, 50.5MB/s] 68% 227M/332M [00:05<00:02, 52.6MB/s] 70% 233M/332M [00:05<00:02, 48.2MB/s] 72% 238M/332M [00:05<00:01, 50.9MB/s] 73% 244M/332M [00:05<00:01, 52.9MB/s] 75% 249M/332M [00:05<00:01, 48.5MB/s] 77% 254M/332M [00:05<00:01, 51.1MB/s] 78% 259M/332M [00:05<00:01, 47.9MB/s] 80% 265M/332M [00:05<00:01, 50.1MB/s] 81% 270M/332M [00:05<00:01, 52.3MB/s] 83% 275M/332M [00:06<00:01, 49.7MB/s] 84% 280M/332M [00:06<00:01, 50.1MB/s] 86% 286M/332M [00:06<00:00, 52.3MB/s] 88% 291M/332M [00:06<00:00, 47.9MB/s] 89% 297M/332M [00:06<00:00, 50.6MB/s] 91% 302M/332M [00:06<00:00, 47.4MB/s] 93% 308M/332M [00:06<00:00, 50.3MB/s] 94% 313M/332M [00:06<00:00, 46.3MB/s] 96% 318M/332M [00:06<00:00, 48.8MB/s] 97% 323M/332M [00:07<00:00, 45.9MB/s] 99% 328M/332M [00:07<00:00, 42.7MB/s]100% 332M/332M [00:07<00:00, 47.7MB/s]
[2021-12-21 22:28:24,115] {subprocess.py:89} INFO - Testing 0 1
[2021-12-21 22:28:24,244] {subprocess.py:89} INFO - /opt/conda/lib/python3.8/site-packages/torch/nn/functional.py:3103: UserWarning: The default behavior for interpolate/upsample with float scale_factor changed in 1.6.0 to align with other frameworks/libraries, and now uses scale_factor directly, instead of relying on the computed output size. If you wish to restore the old behavior, please set recompute_scale_factor=True. See the documentation of nn.Upsample for details.
[2021-12-21 22:28:24,244] {subprocess.py:89} INFO -   warnings.warn("The default behavior for interpolate/upsample with float scale_factor changed "
[2021-12-21 22:28:29,857] {subprocess.py:89} INFO - Testing 1 4
[2021-12-21 22:28:34,833] {subprocess.py:89} INFO - Testing 2 5
[2021-12-21 22:28:39,123] {subprocess.py:89} INFO - Testing 3 6
[2021-12-21 22:28:44,023] {subprocess.py:93} INFO - Command exited with return code 0
[2021-12-21 22:28:44,064] {taskinstance.py:1270} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=super_resolution, execution_date=20211219T000400, start_date=20211221T132810, end_date=20211221T132844
[2021-12-21 22:28:44,113] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-21 22:28:44,167] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
